### PR TITLE
appveyor: cache the build directory so we only build diffs -- appveyor is super slow

### PR DESCRIPTION
**Pull request by fsareshwala**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ environment:
 
 cache:
     - '%CYG_CACHE%'
+    - C:\projects\qfs\build # appveyor is super slow -- cache the build so we only build diffs
```
Comment (mckurt) [inline .appveyor.yml]: Is "C:\projects\qfs\" same as APPVEYOR_BUILD_FOLDER?

Comment (fsareshwala) [inline .appveyor.yml]: Ah yes, let me switch it to that instead.

Comment (mckurt) [inline .appveyor.yml]: nit:
CYG_CACHE: C:\cygwin\var\cache\setup... --> CYGROOT\var\cache...
CYG_BASH: C:\cygwin\bin\bash  --> CYGROOT\bin\bash...
Would that work?

**Comment by mckurt** [comment]
I'll merge this, once builds pass.

**Pull request merged**